bootstrap/storageclass/helmfile/helmfile.yaml

```diff
@@ -19,6 +19,7 @@ environments:
       - "{{ requiredEnv "CK8S_CONFIG_PATH" }}/secrets.yaml"
 
 releases:
+{{ if eq .Values.global.storageClass "nfs-client" }}
```

Comment:
Instead of this `if` statement you should probably use the same approach as in the main helmfile - using the `installed` field, see https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/01-applications.yaml#L121